README.md

```diff
@@ -6,3 +6,8 @@
 
 
 Boston neighborhood reviews
+
+add later in rspec mailer test test WIP:
```

Comment:
This is okay to have in a WIP so that you remember to do it later in the same PR, but it's not good practice to have this living in legacy code on master. Take this out and if you need to add a reminder for yourself to test something later, find a method that isn't merging a to do list to master